src/control/Control.Layers.js

```diff
@@ -192,7 +192,7 @@ export const Layers = Control.extend({
 		    container = this._container = DomUtil.create('div', className),
 		    collapsed = this.options.collapsed;
 
-		// makes this work on IE touch devices by stopping it from firing a mouseout event when the touch is released
+		// makes this work on IE touch devices by stopping it from firing a pointerout event when the touch is released

```

Comment:
Considering this is for Internet Explorer, which we no longer support, perhaps this can be removed?

Reply:
Yes let us remove it with: #9622
https://github.com/Leaflet/Leaflet/commit/e032046119a7f60d0a798472e49e283a29e502da, https://github.com/Leaflet/Leaflet/commit/d18f6249b2bf1db4e0b4be42124fb94303aa99c0,  https://github.com/Leaflet/Leaflet/issues/1539

Reply:
Ok, let's handle it there!